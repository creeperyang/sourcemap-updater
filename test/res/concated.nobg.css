html, body {
    margin: 0;
    padding: 0;
}

body .container {
    display: flex;
    padding: 20px;
}
.box {
    color: #333;
    
}
.box-title {
    transform: scale(
        2
    );
}
button.test {
    
}