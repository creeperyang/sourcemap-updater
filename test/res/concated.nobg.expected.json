{
    "version": 3,
    "sources": [
        "css/simple.css",
        "css/std.css",
        "css/unusual.css"
    ],
    "names": [],
    "mappings": "AAAA;IACA,UAAA;IACA,WAAA;CACA;;AAEA;IACA,cAAA;IACA,cAAA;CACA;ACRA;IACA,YAAA;IACA;CACA;AACA;IACA;;MAEA;CACA;ACRA;IACA,AAEA;CACA",
    "file": "concated.css",
    "sourceRoot": ".."
}